misc/librenms.logrotate

```diff
@@ -1,9 +1,11 @@
 # /etc/logrotate.d/librenms
-/opt/librenms/logs/*.log {
+/opt/librenms/logs/*log {
+    su librenms librenms
     weekly
     rotate 6
     compress
     delaycompress
     missingok
     notifempty
+    create 644 librenms librenms
```

Comment:
You should allow the librenms group to write.  So: 664

Reply:
Agreed :)